dev/devicelab/bin/tasks/module_test.dart

```diff
@@ -226,6 +252,30 @@ Future<void> main() async {
         );
       }
 
+      if (!Platform.isWindows) {
```

Comment:
nit: this check can be removed.

Reply:
The reason is I didn't set the file read only permission at line 54 above for Windows. So, just not checking r-w permission for Windows here and below.

Was thinking verify the file permission change for linux only should be enough.